<ui version="4.0" >
 <class>PrefAdvanced</class>
 <widget class="QWidget" name="PrefAdvanced" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="advanced_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="monitor_aspect_icon" >
           <property name="text" >
            <string>icon</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_8" >
           <property name="text" >
            <string>&amp;Monitor aspect:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>monitoraspect_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyComboBox" name="monitoraspect_combo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>190</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Logs</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="log_smplayer_check" >
            <property name="text" >
             <string>Log &amp;SMPlayer output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="log_mplayer_check" >
            <property name="text" >
             <string>Log MPlayer &amp;output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="log_mplayer_save_check" >
            <property name="text" >
             <string>A&amp;utosave MPlayer log to file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="log_mplayer_save_name" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line8_2" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::HLine</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel1_17" >
            <property name="text" >
             <string>This option is mainly intended for debugging the application.</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2_10" >
            <property name="text" >
             <string>&amp;Filter for SMPlayer logs:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>log_filter_edit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="log_filter_edit" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mplayer_use_window_check" >
         <property name="text" >
          <string>&amp;Run MPlayer in its own window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="not_clear_background_check" >
         <property name="text" >
          <string>&amp;Don't repaint the background of the video window</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="colorkey_label" >
           <property name="text" >
            <string>&amp;Colorkey:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>colorkey_view</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="colorkey_view" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeButton" >
           <property name="text" >
            <string>C&amp;hange...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="shortnames_check" >
         <property name="text" >
          <string>&amp;Pass short filenames (8+3) to MPlayer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Options for MP&amp;layer</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel3_2" >
         <property name="text" >
          <string>Here you can pass extra options to MPlayer.
Write them separated by spaces.
Example: -flip -nosound</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_3" >
           <property name="text" >
            <string>&amp;Options:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>mplayer_args_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_args_edit" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line2" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4_2" >
         <property name="text" >
          <string>You can also pass additional video filters.
Separate them with ",". Do not use spaces!
Example: scale=512:-2,eq2=1.1</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel2_3" >
           <property name="text" >
            <string>V&amp;ideo filters:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>mplayer_vfilters_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_vfilters_edit" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line3" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2" >
         <property name="text" >
          <string>And finally audio filters. Same rule as for video filters.
Example: resample=44100:0:0,volnorm</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel3_3" >
           <property name="text" >
            <string>Audio &amp;filters:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>mplayer_afilters_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_afilters_edit" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage" >
      <attribute name="title" >
       <string>&amp;MPlayer language</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel1_4" >
         <property name="text" >
          <string>SMPlayer needs to read and parse the output of MPlayer and sometimes it relies on English text. If you are using a MPlayer translated into another language, then you need to change the texts that SMPlayer looks for. (Technically you should enter regular expressions)&lt;br>&lt;br>
The drop-down lists may provide already made regular expression for several languages.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="textLabel2_12" >
         <property name="text" >
          <string>&amp;End of file:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>endoffile_combo</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="textLabel3_6" >
         <property name="text" >
          <string>&amp;No video:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>novideo_combo</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="MyComboBox" name="endoffile_combo" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="MyComboBox" name="novideo_combo" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>advanced_tab</tabstop>
  <tabstop>monitoraspect_combo</tabstop>
  <tabstop>mplayer_use_window_check</tabstop>
  <tabstop>colorkey_view</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>not_clear_background_check</tabstop>
  <tabstop>endoffile_combo</tabstop>
  <tabstop>novideo_combo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>log_mplayer_save_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>log_mplayer_save_name</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
